raiden/message_handler.py

```diff
@@ -155,34 +153,14 @@ def message_refundtransfer(self, message):
             message.lock.hashlock,
         )
 
-        identifier = message.identifier
-        token_address = message.token
-        target = message.target
-        amount = message.lock.amount
-        expiration = message.lock.expiration
-        hashlock = message.lock.hashlock
-
-        manager = self.raiden.identifier_to_statemanagers[identifier]
-
-        if isinstance(manager.current_state, InitiatorState):
```

Comment:
So the main functionality change of this PR is that we now always take the `initiator` of the RefundTransfer from the `message` instead of doing these 2 checks depending on the corresponding state manager's current state. Correct?